### DISCUSSION TITLE
Anycable Go server not consuming payload from redis.

### DISCUSSION THREAD

**Discussion by akashsuresh** [anycable/anycable#133] [Q&A]
I have setup a chat application on rails using anycable. Now I am successfully able to receive messages from the client, however when push anything to the channel, I get a success from the job that I have created

`I, [2021-06-05T14:33:49.104566 #349972]  INFO -- : [ActiveJob] [MessageBroadcastJob] [6b6556c5-3944-44cd-84b9-cf2778e1d86e] Performed MessageBroadcastJob (Job ID: 6b6556c5-3944-44cd-84b9-cf2778e1d86e) from Async(default) in 1.84ms`

However I do not receive the message on the client side. I connected to the redis channel __anycable__ manually and I was able to consume messages from the channel.

What could the issue be?

**Comment by palkan** [comment]
Make sure you:
- Use `any_cable` as your PubSub adapter in `cable.yml`
- Use the same `redis_channel` in `cable.yml` and when starting anycable-go.

If anycable-go cannot connect to a Redis server, it prints errors and eventually crashes. Since you're not seeing it crashing, it successfully connects to a Redis server; thus, there is likely a channel misconfiguration.
